docs/getting_started/install_using_ddev.md

```diff
@@ -0,0 +1,410 @@
+---
+description: Install Ibexa DXP with Docker and DDEV to use it for development.
+---
+
+# Install using DDEV
+
+This guide provides a step-by-step walkthrough of installing [[= product_name =]] using [DDEV](https://ddev.com/).
+DDEV is an open-source tool that simplifies the process of setting up local PHP development environments.
+
+## Requirements
+
+Before you start the installation, ensure you have the following software installed:
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [DDEV](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/)
+
+## Installation
+
+### 1. Create a DDEV project directory
+
+Start by creating a directory for your DDEV project by using the following command to create a new directory:
+
+```bash
+mkdir my-ddev-project && cd my-ddev-project
+```
+
+Replace `my-ddev-project` with your desired directory name.
+
+### 2. Configure DDEV
+
+#### Configure PHP version and document root
+
+Next, configure your DDEV environment using the command below:
+
+```bash
+ddev config --project-type=php --php-version 8.1 --docroot=public --create-docroot
+```
+
+This command sets the project type to PHP, the PHP version to 8.1, the document root to `public` directory, and creates the document root.
+
+#### Switch to Apache (optional)
+
+By default, DDEV uses Nginx.
+
+To use Apache instead, run the following command:
+
+```bash
+ddev config --webserver-type=apache-fpm
+```
+
+#### Use another database type (optional)
+
+By default, DDEV uses MariaDB.
+
+To use PostgreSQL instead, run the following command:
+
+```bash
+ddev config --database=postgres:15
+```
+
+To use MySQL instead, run the following command:
+
+```bash
+ddev config --database=mysql:8.0
+```
+
+Other version of MariaDB, Mysql or PostgreSQL can be used. See [DDEV database types documentation](https://ddev.readthedocs.io/en/latest/users/extend/database-types/) for available version ranges.
+
+#### Configure database connection
+
+Now, configure the database connection for your Ibexa DXP project. Depending on your database of choice (MySQL or PostgreSQL), use the appropriate command below.
+
+!!! note
+
+    Those commands will set a `DATABASE_URL` environment variable inside the container which overrides [the variable from `.env`](install_ibexa_dxp.md#change-installation-parameters).
```

Comment:
```suggestion
    Those commands set a `DATABASE_URL` environment variable inside the container which overrides [the variable from `.env`](install_ibexa_dxp.md#change-installation-parameters).
```

Reply:
There is a bug with Vale here. If I remove the indentation, I obtain the expected `Ibexa.FutureTense` warning "Use present tense instead of future."